windows/docker-compose.yml

```diff
@@ -45,6 +46,9 @@ services:
       - SITE_ROOT=api
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
+      - "${DEV_CONF_PATH:-/dev/null}:/openimis-dev"
     depends_on:
       database:
         condition: service_healthy
```

Comment:
please uncomment the frontend and add the dev_path (dev_conf_path is missleading)